java/templates/deployment.yaml

```diff
@@ -4,13 +4,9 @@ kind: Deployment
 metadata:
   name:  {{ template "hmcts.releaseName" . }}
   labels:
-    app.kubernetes.io/name: {{ template "hmcts.releaseName" . }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/instance: {{ template "hmcts.releaseName" . }}
-    {{- if .Values.aadIdentityName }}
-    aadpodidbinding: {{ .Values.aadIdentityName }}
-    {{- end }}
+    {{- ( include "java.labels" . ) | indent 4 }}
```

Comment:
Can we resuse this in other places like configmap, svc?

Reply:
probably 😉, although dunno if we want the prometheus annotations etc everywhere